### DISCUSSION TITLE
404 Help Links on Admin Endpoints

### DISCUSSION THREAD

**Discussion by nCubed** [keygen-sh/community#37] [Issue]
On the admin portal, the two help links (Url, Subscriptions) for new Endpoints are 404: https://app.keygen.sh/webhook-endpoints/new

**Comment by ezekg** [comment]
Thanks for the report. This has been fixed!

**Comment by nCubed** [comment]
More 404's on the admin portal. 

Settings > Edit Account: each of the "view attribute docs" links are dead 404's.
